src/import-data/ImportDataDestination.ts

```diff
@@ -189,13 +189,14 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
     buildDestinationWorkspaceFilter(importRequest, { requiredAuthorizationDomain })
   );
 
-  // disable import into existing workspaces if data is marked as protected but no protected workspaces are available
-  const disableExportIntoExisting = isProtectedData && workspacesToImportTo.length === 0;
+  // disable import into existing workspaces if no suitable workspaces are available
+  // TODO: this should apply no matter the exact import requirements
+  const disableExportIntoExisting = importRequiresSecurityMonitoring && workspacesToImportTo.length === 0;
```

Comment:
Since there are multiple requirements that may limit the available destination workspaces, this shouldn't care about `importRequiresSecurityMonitoring` specifically. I tried updating this, but some unrelated tests broke so I left it to limit the scope of this PR.